fiftyone/server/routes/sample.py

```
@@ -645,10 +645,113 @@ async def patch(self, request: Request, data: List[dict]) -> JSONResponse:
+            )
+
+        dataset = get_dataset(dataset_id)
+        sample = dataset[sample_id]
+
+        try:
+            label_field = sample[field]
+        except KeyError:
+            raise HTTPException(
+                status_code=404,
+                detail=f"Field '{field}' not found on sample",
+            )
```

Ruff (0.15.11)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)